packages/components/src/Combobox/components/ComboboxContent/ComboboxContent.tsx

```diff
@@ -203,7 +323,10 @@ function useComboboxContent(options: ComboboxOption[]): {
     }
   }, "Escape");
 
-  const popperRef = useRef<HTMLDivElement>(null);
+  useRefocusOnActivator(open);
```

Comment:
so I don't love having this in here. it's arguably a lot more accessibility related than it is generic behavior of Combobox. I tried moving it into the `useComboboxAccessibility` but...

the problem is that if I call this **after** `useFocusTrap` then the wrong `document.activeElement` gets used as you can see here 

<img width="769" alt="Components___Combobox___Combobox___Web_-_Clear_Selection_⋅_Storybook" src="https://github.com/GetJobber/atlantis/assets/11843215/3b4df5c7-83b0-42a6-882b-e9696bb64ec2">

and then it just doesn't work because the focus doesn't return to the activator it goes into the now closed dialog :| 

now I'm sure there's a way to make this order independent, but I'm not sure I want to fix that right now - so lemme know what you think about this setup. I could move all this `popperRef` and `useFocusTrap` into the other hook

same with the `useOnKeyDown` honestly too. that's accessibility focused.